meson.build

```diff
@@ -321,9 +321,30 @@ cc = meson.get_compiler('c')
 meson_build_sh = find_program('tools/meson-build.sh')
 
 want_tests = get_option('tests')
-slow_tests = want_tests != 'false' and get_option('slow-tests')
-fuzz_tests = want_tests != 'false' and get_option('fuzz-tests')
+if want_tests == 'false'
+        slow_tests = false
+        fuzz_tests = false
+else
+        slow_tests = get_option('slow-tests')
+        if slow_tests == 'auto'
+                slow_tests = conf.get('BUILD_MODE_DEVELOPER') == 1
```

Comment:
Hmm, so I think this is backwards. It's fine to run the slow tests for release builds, which are with optimization enabled and take a long time anyway and we care about as-complete coverage as possible. But for development, I want the opposite and I think this should be the default: quick builds without having to tweak obscure options.

E.g. in Fedora, we don't enable slow tests in package builds. This seems to be a mistake. We also don't enable fuzz tests, which might be a mistake too. This is precisely the scenario where one would want to have wide test coverage.

Reply:
In the release mode, usually used in the distro packagers, they do not manually run meson but run through e.g. spec file or so. Hence, I do not have any strong opinion about we should enable or disable them on release mode.

In the development mode, I think we should enable them, to avoid to introduce regressions, any new memleak or so. Faster development does not have priority than the safer development.

But anyway, I dropped the change. Now the test mode are shown in the summary, hence we can easily see the testing modes are enabled or not. And we can choose preferred mode for each test option by each developer.